src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/Histogram.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip
+} from 'recharts';
+import PropTypes from 'prop-types';
+
+//TODO - add legend, auto-scaling, sorting, etc
```

Comment:
future ticket.

Reply:
Would you like me to create a ticket for this? Or have you created one?

Reply:
I'll add one